frontend/src/components/BookingForm.js

```diff
@@ -113,16 +115,38 @@ const BookingForm = () => {
     return dayjs(date).format("YYYY-MM-DD")
   }
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault()
+
     if (!selectedCheckOutDate) {
       alert("Please select an end date before submitting")
       return
     }
-    console.log(
-      `Selected Date Range:\n${selectedCheckInDate} - \n${selectedCheckOutDate}`
-    )
-    // need to add booking to firestore
+
+    const checkInFormatted = selectedCheckInDate.toDate()
+    const checkOutFormatted = selectedCheckOutDate.toDate()
+
+    try {
+      const booking = {
```

Comment:
Hmm, should we make this consistent with the [Architecture spec](https://github.com/UoaWDCC/uasc-web/blob/main/frontend/ARCHITECTURE.md#bookings-collection)?